client/src/app/core/services/http.service.ts

```diff
@@ -60,21 +60,26 @@ export class HttpService {
         method: HTTPMethod,
         data?: any,
         queryParams?: QueryParams,
-        customHeader?: HttpHeaders
+        customHeader?: HttpHeaders,
+        responseType?: string
```

Comment:
What is allowed to provide here? Does somewhere exist a list of content types? For now, you may lock this to all needed types. Maybe define a `type ContentType = 'json' | '...' | ...;` on top of the file.

Reply:
string is the correct type, there might be a lot of response types.
I set it in the documentation, sry about that.
https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/responseType